octodns/provider/cloudflare.py

```diff
@@ -222,7 +236,14 @@ def _record_for(self, zone, name, _type, records, lenient):
             data_for = getattr(self, '_data_for_{}'.format(_type))
             data = data_for(_type, records)
 
-        return Record.new(zone, name, data, source=self, lenient=lenient)
+        record = Record.new(zone, name, data, source=self, lenient=lenient)
+
+        if records[0].get('proxiable', False):
```

Comment:
What's the `proxiable` flag here? Would this run into problems if Cloudflare added a new type that had this true, but `_PROXIABLE_RECORD_TYPES` didn't include it?